experimental/Experimental.jl

```diff
@@ -47,6 +47,8 @@ for pkg in exppkgs
   include(joinpath(expdir, pkg, "src", "$pkg.jl"))
 end
 
+# force trigger recompile when folder changes
+include_dependency(".")
```

Comment:
@lkastner : This should probably be reverted, right? I'm just trying to untangle the mess created by my rebase...